app/helpers/application_helper/button/cockpit_console.rb

```diff
@@ -17,8 +28,8 @@ def on?
     @record.ready_condition_status == 'True' if @record.respond_to?(:ready_condition_status) # Container status
   end
 
-  def platform_supported?(record_type)
-    if record_type == 'VM'
+  def platform_supported?
+    if @record.respond_to?(:current_state)
```

Comment:
this change is intended? (does not look translation related)

Reply:
Yes, intended. In the code above, the `record_type` argument passed in would either
be `_('VM')` or `_('Container Node')`. Now that wouldn't really work in non-English locales,
would it?

Reply:
Makes sense. Thx!